app/models/pagination.rb

```diff
@@ -0,0 +1,13 @@
+class Pagination < ApplicationStruct
+  attribute :current_page, Types::Params::Integer.optional.default(1)
```

Comment:
If we have a default page here for current page, can we somehow remove the one from the `app/controllers/application_controller.rb` L33 seems like a duplication? 🤔